fastlane/Fastfile

```diff
@@ -1028,7 +1028,10 @@ def read_version_from_config
 end
 
 def ensure_sentry_installed
-    UI.user_error('sentry-cli not installed') unless sh('command -v sentry-cli')
+  # This is an action provided by the Sentry Fastlane plugin that verifies the
+  # CLI is installed and its version is compatible with the plugin's
+  # expectation.
+  sentry_check_cli_installed
 end
```

Comment:
That's quite a nice and welcome change :)

At that point though I'm not sure it's worth keeping a `def` dedicated method that ends up just calling the action though, as opposed to calling `sentry_check_cli_installed` directly everywhere we previously called `ensure_sentry_installed` in the rest of the `Fastfile`?